package.json

```diff
@@ -77,14 +76,14 @@
     "request": "^2.74.0",
     "requestretry": "^1.12.0",
     "screwdriver-config-parser": "^3.0.0",
-    "screwdriver-data-schema": "^14.0.2",
+    "screwdriver-data-schema": "^14.1.0",
     "screwdriver-datastore-dynamodb": "^3.0.0",
     "screwdriver-datastore-sequelize": "^1.0.2",
     "screwdriver-executor-docker": "^1.1.2",
     "screwdriver-executor-k8s": "^9.0.0",
     "screwdriver-models": "^16.0.0",
-    "screwdriver-scm-bitbucket": "^2.0.2",
-    "screwdriver-scm-github": "^3.0.0",
+    "screwdriver-scm-bitbucket": "^2.1.0",
+    "screwdriver-scm-github": "^4.1.0",
```

Comment:
I think we should do this incrementally. I don't want to merge that PR just yet.